lib/ansible/modules/cloud/vmware/vmware_datastore_cluster.py

```diff
@@ -44,6 +47,22 @@
       - If the datastore cluster should be present or absent.
       choices: [ present, absent ]
       default: present
+    folder:
+      description:
+      - Destination folder, absolute path to place datastore cluster in.
+      - The folder should include the datacenter.
+      - This parameter is case sensitive.
+      - This parameter is optional, if C(datacenter) is provided.
+      - 'Examples:'
```

Comment:
This is a lot of examples. Could you trim it back to 3-5?

Reply:
I think each example denotes different vCenter environment and will help user to write a playbook without trial-error. I want to keep this way.